### PR TITLE
Fix : Path tracker causes app to crash #362 (PR 3, changes after review)

### PR DESCRIPTION
**Pull request by reverie-ss**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

@therajanmaurya 
**This is the final PR after review. The last PR #399 has been closed**

- [x] Changes mentioned in the comments of #399 has been done
- [x] All these commits have been squashed into one:
        - Fix : Path tracker causes app to crash #362 (2)
        - Fix : Path tracker causes app to crash #362 (made changes after reviewed)
        - Fix : Path tracker causes app to crash #362 (PR 3)

### CODE REVIEW

**Comment by reverie-ss** [comment]
@therajanmaurya 
I figured out that #361 has the same issue of #362 .
This fix #418 can be implemented at both places.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/PathTrackingActivity.java [hunk 1]
```diff
@@ -40,22 +48,21 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
-            @Override
-            public void onClick(View view) {
-                service.stopLogging();
-            }
-        });
         showBackButton();
     }
 
+    @OnClick(R.id.start)
+    public void setStart() {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/PathTrackingActivity.java]: This method name should be like ```onClickStartTracking()```

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/PathTrackingActivity.java [hunk 2]
```diff
@@ -40,22 +48,21 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
...
+        if (checkPermissionAndRequest()) {
+            service.startLogging();
+        }
+    }
+
+    @OnClick(R.id.stop)
+    public void setStop() {
```
Comment (therajanmaurya) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/activity/PathTrackingActivity.java]: This method name should be like ```onClickStopTracking()```

**Comment by therajanmaurya** [comment]
Please look the above changes and also change Button variable name from ```stop``` to ```btn_stop_tracking`` and ```start``` to ```btn_start_tracking``` and ```service``` to ```locationService```.

Please make the changes and let me know for the final review.

**Comment by therajanmaurya** [comment]
Hi, @ishan1604 This PR looks good Please give final review and merge.

**Comment by ishan1604** [comment]
@therajanmaurya Why are we using snake case for naming fields in Java classes ? 😟

**Comment by therajanmaurya** [comment]
Sorry, I fotget and thinking the same but confused, what's wrong here.
We are following https://github.com/openMF/android-client/wiki/Code-style-conventions#51-id-naming but little confused.
@ishan1604 is that we have to change ```btn_stop_tracking``` to ```btn_stopTacking```
and ```btn_start_tracking``` to ```btn_startTracking```

**Comment by reverie-ss** [comment]
@therajanmaurya 
should i make the changes?

**Comment by therajanmaurya** [comment]
@reverie-ss Change it to ```btnStartTracking``` and ```btnStopTacking```

**Comment by therajanmaurya** [comment]
This feature was not implemented properly. Location was going nowhere. 
Refactor and fix in #478

**Pull request closed without merging**